app/infrastructure/broadcaster.py

```
@@ -0,0 +1,45 @@
+import asyncio
+from typing import Optional
+
+
+class MetricsBroadcaster:
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)